debian/control

```diff
@@ -2,7 +2,7 @@ Source: kodi-pvr-iptvsimple
 Priority: extra
 Maintainer: Anton Fedchin <anightik@gmail.com>
 Build-Depends: debhelper (>= 9.0.0), cmake, libkodiplatform-dev (>= 16.0.0),
-               kodi-addon-dev, librapidxml-dev, zlib1g-dev
+               kodi-addon-dev, librapidxml-dev, zlib1g-dev, openssl
```

Comment:
openssl is the openssl binary, you need the openssl library and headers (libssl-dev)

Reply:
Ah, ok, thanks